src/bioutils/assemblies.py

```diff
@@ -104,20 +110,20 @@ def get_assemblies(names=[]):
             are the dictionaries of assembly data as described in ``get_assembly()``.
 
     Examples:
-        >>> assemblies = get_assemblies(names=['GRCh37.p13'])
-        >>> assy = assemblies['GRCh37.p13']
+        >>> assemblies = get_assemblies(names=["GRCh37.p13"])
+        >>> assy = assemblies["GRCh37.p13"]
 
         >>> assemblies = get_assemblies()
-        >>> 'GRCh38.p2' in assemblies
+        >>> "GRCh38.p2" in assemblies
         True
-    """
 
-    if names == []:
+    """
+    if names is None:
```

Comment:
Do we also need to handle the case when names is an empty list?

```suggestion
    if not names:
```

Reply:
I don't remember why I would've made this change. cause for reflection

Reply:
Ok, I think this was some fixes related to type changes. Basically, a bunch of functions were implementing "optional" args differently, and I went through and made them `Optional` in the typing sense and had them default to None.

Here, I think the previous behavior is kind of silly (if `names` is used to filter the metadata that you retrieve, then presumably an empty list should return empty results) but for the sake of continuity I think we should probably stick with the previous behavior